src/test/scala/kanaloa/reactive/dispatcher/IntegrationTests.scala

```diff
@@ -84,6 +84,47 @@ class PushingDispatcherIntegration extends IntegrationSpec {
 
 }
 
+class MinimalPushingDispatcherIntegration extends IntegrationSpec {
+
+  "can process through a large number of messages" in new TestScope {
+
+    val backend = TestActorRef[SimpleBackend]
+
+    val pd = system.actorOf(PushingDispatcher.props(
+      "test-pushing",
+      Backend(backend),
+      ConfigFactory.parseString(
+        s"""
+          |kanaloa.dispatchers.test-pushing {
+          |  workerPool {
+          |    startingPoolSize = 8
+          |  }
+          |  autoScaling {
+          |    enabled = off
+          |  }
+          |  backPressure {
+          |    enabled = off
+          |  }
+          |  circuitBreaker {
+          |    enabled = off
+          |  }
+          |}
+        """.stripMargin
```

Comment:
Since the `stripMargin`s were removed from DispatcherSpec, maybe they should be removed in this file also, for consistency